### DISCUSSION TITLE
Some translations languages are not listed in language settings preferences listbox (in my case i look for italian)

### DISCUSSION THREAD

**Discussion by marcobra** [pawelsalawa/sqlitestudio#4182] [General]
### Details
Here you describe your problem details, or proposal for enhancement.
We cannot see many languages listed in pref → lang setting → listbox 
I'm sqlitestudio translator and it is almost at 100% translated in italian
### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.
see above...
### Operating system
This applies only to bug reports. If this is a feature request, delete this section.
Linux ubuntu
### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.
SqliteStudio 3.3.3

**Comment by pawelsalawa** [comment]
Only translations packaged in the released version are available in this list.
